Services/WindowServer/MenuItem.h

```diff
@@ -77,6 +77,7 @@ class MenuItem {
     void set_submenu_id(int submenu_id) { m_submenu_id = submenu_id; }
 
     Menu* submenu();
+    const Menu* submenu() const;
```

Comment:
Where's the definition?

Reply:
brainfart, might go sleep now... suprised this builds and runs okay to be honest.